src/main/java/assets/mixxy.js

```diff
@@ -1,3 +1,8 @@
 /**
  * Created by MURDERBUS on 4/7/16.
  */
+$(document).ready(function(){
+    $('.featured-comics').slick({
```

Comment:
If there's only going to be one `featured-comics` element, wouldn't this best be an ID?  (`#featured-comics`)  Better performance that way (only have to find one element instead of theoretically many)